extension/duckdb/src/duckdb_scan.cpp

```diff
@@ -28,8 +28,10 @@ DuckDBScanBindData::DuckDBScanBindData(std::string query,
 }
 
 std::unique_ptr<TableFuncBindData> DuckDBScanBindData::copy() const {
-    return std::make_unique<DuckDBScanBindData>(query, LogicalType::copy(columnTypes), columnNames,
-        connector);
+    auto result = std::make_unique<DuckDBScanBindData>(query, LogicalType::copy(columnTypes),
```

Comment:
Technically u should have a copy constructor handles copy. This style has the problem that if add new field in the base class it will not be copied automatically.

Reply:
Actually u can ignore this comment. I think @mxwli's PR also fixes this.